censustree/arbotree/wrapper.go

```diff
@@ -73,7 +73,7 @@ func (t *Tree) TypeString() string {
 // to choose the hash function to be used in the Tree.
 func (t *Tree) Init(name, storageDir string) error {
 	dbDir := filepath.Join(storageDir, "arbotree.db."+strings.TrimSpace(name))
-	database, err := db.NewBadgerDB(dbDir)
+	database, err := badgerdb.NewBadgerDB(badgerdb.Options{Path: dbDir})
```

Comment:
nit: if you rename the package to badgerdb, might as well do `badgerdb.New` to prevent stutter